UPGRADE-1.14.md

```diff
@@ -639,98 +632,97 @@
+|-----------------------------------------------------------------------|-------------------------------------------------------------------------|
+| `Sylius\Bundle\AdminBundle\Provider\StatisticsDataProvider`           | `Sylius\Component\Core\Statistics\Provider\StatisticsProvider`          |
+| `Sylius\Bundle\AdminBundle\Provider\StatisticsDataProviderInterface`  | `Sylius\Component\Core\Statistics\Provider\StatisticsProviderInterface` |
+| `Sylius\Bundle\CoreBundle\Twig\StateMachineExtension`                 | `Sylius\Abstraction\StateMachine\Twig\StateMachineExtension`            |
 
-1. The class `Sylius\Bundle\CoreBundle\Form\EventSubscriber\AddUserFormSubscriber` has been deprecated and will be removed in Sylius 2.0.
+| Class/Interface                                                       | Will be replaced with                                                   |
+|-----------------------------------------------------------------------|-------------------------------------------------------------------------|
+| `Sylius\Bundle\AdminBundle\EventListener\ResourceDeleteSubscriber`    | `Sylius\Bundle\AdminBundle\EventListener\ResourceDeleteListener`        |
+| `Sylius\Bundle\CoreBundle\Theme\ChannelBasedThemeContext`             | `Sylius\Bundle\ShopBundle\Theme\ChannelBasedThemeContext`               |
+| `Sylius\Bundle\CoreBundle\Form\EventSubscriber\AddUserFormSubscriber` | `Sylius\Bundle\AdminBundle\Form\EventSubscriber\AddUserFormSubscriber`  |
 
-1. The class `Sylius\Bundle\AdminBundle\EventListener\ResourceDeleteSubscriber` has been deprecated and will be removed in Sylius 2.0.
-   It will be replaced with the `ResourceDeleteListener`.
-
-1. The following exceptions have been deprecated and will be removed in Sylius 2.0:
-    - `Sylius\Bundle\ApiBundle\Exception\CannotRemoveCurrentlyLoggedInUser`
-    - `Sylius\Bundle\ApiBundle\Exception\ChannelCannotBeRemoved`
-    - `Sylius\Bundle\ApiBundle\Exception\PaymentMethodCannotBeRemoved`
-    - `Sylius\Bundle\ApiBundle\Exception\ProductAttributeCannotBeRemoved`
-    - `Sylius\Bundle\ApiBundle\Exception\ProductCannotBeRemoved`
-    - `Sylius\Bundle\ApiBundle\Exception\ProductVariantCannotBeRemoved`
-    - `Sylius\Bundle\ApiBundle\Exception\PromotionCannotBeRemoved`
-    - `Sylius\Bundle\ApiBundle\Exception\PromotionCouponCannotBeRemoved`
-    - `Sylius\Bundle\ApiBundle\Exception\ProvinceCannotBeRemoved`
-    - `Sylius\Bundle\ApiBundle\Exception\ShippingMethodCannotBeRemoved`
-    - `Sylius\Bundle\ApiBundle\Exception\TaxonCannotBeRemoved`
-    - `Sylius\Bundle\ApiBundle\Exception\ZoneCannotBeRemoved`
-
-    All of these exceptions will be replaced with the `Sylius\Component\Core\Exception\ResourceDeleteException` exception.
+| Class/Interface                                                                                                        |
```

Comment:
As in this table is only a single column, I would be probably for not using a table here 🤔

Reply:
:+1: a simple list would be better